packages/core/src/messagingProtocols/mls/E2EIdentityService/E2EIServiceExternal.ts

```diff
@@ -205,9 +216,9 @@ export class E2EIServiceExternal extends TypedEventEmitter<Events> {
    *
    * Both must be registered before the first enrollment.
    */
-  public async registerServerCertificates(): Promise<void> {
+  private async registerServerCertificates(): Promise<void> {
     const ROOT_CA_KEY = 'e2ei_root-registered';
-    const store = LocalStorageStore(ROOT_CA_KEY);
+    const store = LocalStorageStore(this.coreDatabase.name);
```

Comment:
We need to make sure that this key is unique per user (in case someone logs in to another account on the same browser)